src/fides/api/util/tcf/tcf_experience_contents.py

```diff
@@ -422,9 +438,47 @@ def build_purpose_or_feature_section_and_update_vendor_map(
                 privacy_declaration_row=privacy_declaration_row,
             )
 
+        # Separately, add AC Vendor to Vendor Consent Map if applicable
+        add_ac_vendor_to_vendor_consent_map(
+            vendor_map=vendor_map,
+            tcf_vendor_component_type=tcf_vendor_component_type,
+            privacy_declaration_row=privacy_declaration_row,
+        )
```

Comment:
This adds AC vendors to the Vendor Consents section if they weren't there already. What it *doesn't do* is prevent this from showing up in other sections.  If the AC vendor had privacy declarations with certain data uses and appropriate legal bases it could show up elsewhere too.  It didn't seem like I needed to *prevent* that specifically, and this seems more theoretical than something we'd actually encounter

Reply:
yup I agree—that makes sense to me!

Reply:
agreed, seems like the right way to handle things, at least for now. we'll see if we need to circle back if people start adding declarations to these systems and it hits unexpected edge cases here 👍